atlassian_jwt_auth/signer.py

```diff
@@ -36,18 +40,22 @@ def _generate_claims(self, audience, **kwargs):
             'nbf': now,
             'sub': self.issuer,
         }
-        claims.update(kwargs.get('additional_claims', {}))
+        if additional_claims is not None:
+            claims.update(additional_claims)
         return claims
 
     def _now(self):
         return datetime.datetime.utcnow()
 
-    def generate_jwt(self, audience, **kwargs):
+    def generate_jwt(self, audience, additional_claims=None):
```

Comment:
Switching this to have `additional_claims` instead of `kwargs` here certainly does seem like an improvement here.